applications/asset_tracker_v2/tests/lwm2m_integration/src/lwm2m_integration_test.c

```diff
@@ -327,86 +328,51 @@ void test_lwm2m_integration_network_error(void)
 	TEST_ASSERT_EQUAL(CLOUD_WRAP_EVT_ERROR, last_cb_type);
 }
 
-void test_lwm2m_integration_fota_result_get(void)
-{
-	/* Expect the FOTA update result to be retrieved for any update in FOTA state. */
-	__cmock_lwm2m_get_u8_ExpectAndReturn(&LWM2M_OBJ(5, 0, 5), NULL, 0);
-	__cmock_lwm2m_get_u8_IgnoreArg_value();
-
-	__cmock_lwm2m_get_u8_ExpectAndReturn(&LWM2M_OBJ(5, 0, 5), NULL, 0);
-	__cmock_lwm2m_get_u8_IgnoreArg_value();
-
-	__cmock_lwm2m_get_u8_ExpectAndReturn(&LWM2M_OBJ(5, 0, 5), NULL, 0);
-	__cmock_lwm2m_get_u8_IgnoreArg_value();
-
-	__cmock_lwm2m_get_u8_ExpectAndReturn(&LWM2M_OBJ(5, 0, 5), NULL, 0);
-	__cmock_lwm2m_get_u8_IgnoreArg_value();
-	__cmock_lwm2m_firmware_set_update_state_cb_Expect(NULL);
-
-	firmware_update_state_cb(STATE_IDLE);
-	firmware_update_state_cb(STATE_DOWNLOADING);
-	firmware_update_state_cb(STATE_DOWNLOADED);
-	firmware_update_state_cb(STATE_UPDATING);
-}
-
-void test_lwm2m_integration_fota_result_get_error(void)
-{
-	__cmock_lwm2m_get_u8_ExpectAnyArgsAndReturn(-1);
-
-	firmware_update_state_cb(STATE_DOWNLOADING);
-	TEST_ASSERT_EQUAL(CLOUD_WRAP_EVT_ERROR, last_cb_type);
-}
-
 void test_lwm2m_integration_fota_error(void)
 {
-	__cmock_lwm2m_get_u8_IgnoreAndReturn(0);
+	struct lwm2m_fota_event event = { .id = LWM2M_FOTA_UPDATE_ERROR,
```

Comment:
Nit: We typically structure these like this:

Just to align with the rest of the app

```
	struct lwm2m_fota_event event = {
                .id = LWM2M_FOTA_UPDATE_ERROR,
		.failure.obj_inst_id = 0,
		.failure.update_failure = 10
         };
```

Reply:
I was using Zephyr code style.

Reply:
OK I do manually that